### PR TITLE
Add Windows.System.Powershell.PSReadline

### PR DESCRIPTION
**Pull request by mgreen27**
Added PSReadline, this is a handy artefact when investigating interactive powershell attacks.

name: Windows.System.Powershell.PSReadline
description: |
  This Artifact will search and extract lines from PSReadline history file.

  Powershell is commonly used by attackers across all stages of the attack 
  lifecycle. The PSReadline module is responsible for command history and from 
  Powershell 5 on Windows 10 default configuration saves a copy of the console 
  history to disk.
  
  There are several parameter's available for search leveraging regex.
  - SearchStrings enables regex search over a PSReadline line.
  - StringWhiteList enables a regex whitelist for results.
  - UserRegex enables a regex search on Username

### CODE REVIEW

File: artifacts/definitions/Windows/EventLogs/PowershellModule.yaml [hunk 1]
```diff
@@ -35,21 +35,14 @@ parameters:
     description: "regex search over Payload text field."
   - name: SearchVSS
     description: "Add VSS into query."
-    type: bool
-  - name: LogLevelMap
```
Comment (scudette) [inline artifacts/definitions/Windows/EventLogs/PowershellModule.yaml:39]: Why is this removed? It is needed below.

Comment (mgreen27) [reply artifacts/definitions/Windows/EventLogs/PowershellModule.yaml:39]: @scudette For the PowershellModule content I took the PowershellScriptblock and copy pasta'd.
Module Logging doesnt have that log level setting so I removed it from the query but noticed I left it in the parameter definitions. I missed the VQL which was what you noticed here.  I just added a commit to update and remove the VQL.
